crates/torii/graphql/src/mapping.rs

```diff
@@ -0,0 +1,88 @@
+use async_graphql::dynamic::TypeRef;
+use async_graphql::Name;
+use dojo_types::primitive::Primitive;
+use indexmap::IndexMap;
+use lazy_static::lazy_static;
+
+use crate::types::{GraphqlType, TypeData, TypeMapping};
+
+lazy_static! {
+    pub static ref ENTITY_TYPE_MAPPING: TypeMapping = IndexMap::from([
+        (Name::new("id"), TypeData::Simple(TypeRef::named(TypeRef::ID))),
+        (Name::new("keys"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+        (Name::new("model_names"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+        (Name::new("event_id"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+        (
+            Name::new("created_at"),
+            TypeData::Simple(TypeRef::named(GraphqlType::DateTime.to_string())),
+        ),
+        (
+            Name::new("updated_at"),
+            TypeData::Simple(TypeRef::named(GraphqlType::DateTime.to_string())),
+        ),
+    ]);
+    pub static ref EVENT_TYPE_MAPPING: TypeMapping = IndexMap::from([
+        (Name::new("id"), TypeData::Simple(TypeRef::named(TypeRef::ID))),
+        (Name::new("keys"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+        (Name::new("data"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+        (
+            Name::new("created_at"),
+            TypeData::Simple(TypeRef::named(GraphqlType::DateTime.to_string())),
+        ),
+        (Name::new("transaction_hash"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+    ]);
+    pub static ref MODEL_TYPE_MAPPING: TypeMapping = IndexMap::from([
+        (Name::new("id"), TypeData::Simple(TypeRef::named(TypeRef::ID))),
+        (Name::new("name"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+        (
+            Name::new("class_hash"),
+            TypeData::Simple(TypeRef::named(Primitive::Felt252(None).to_string())),
+        ),
+        (
+            Name::new("transaction_hash"),
+            TypeData::Simple(TypeRef::named(Primitive::Felt252(None).to_string())),
+        ),
+        (
+            Name::new("created_at"),
+            TypeData::Simple(TypeRef::named(GraphqlType::DateTime.to_string())),
+        ),
+    ]);
+    pub static ref SYSTEM_CALL_TYPE_MAPPING: TypeMapping = IndexMap::from([
+        (Name::new("id"), TypeData::Simple(TypeRef::named(TypeRef::ID))),
+        (Name::new("transaction_hash"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+        (Name::new("data"), TypeData::Simple(TypeRef::named(TypeRef::STRING))),
+        (Name::new("system_id"), TypeData::Simple(TypeRef::named(TypeRef::ID))),
```

Comment:
this should probably be `contract_address` now, since we are removing systems

Reply:
Gotcha, can we address this in another PR that is specifically for system call indexing? Will need to update db schema and relationship queries as well

Reply:
Sure np!